db/sql/init001.sql

```diff
@@ -0,0 +1,8 @@
+create extension if not exists "uuid-ossp";
+CREATE TABLE email_account (
+  id          uuid      NOT NULL UNIQUE PRIMARY KEY DEFAULT uuid_generate_v4(),
+  email       text      NOT NULL,
+  has_pubkey  bool      DEFAULT false,
```

Comment:
@ajvb Hmm, is this necessary? It'd be nice if the DB and GPG keyring couldn't get out of sync with each other (in cases where an email account is created, then I import their PGP key, and then try to email them... and it accidentally gets sent unencrypted).

Reply:
I agree, should just have it check of the existence of a key on the keyring.